roles/common/tasks/packages.yml

```diff
@@ -11,6 +11,31 @@
     - install
     - packages
 
+- name: Install packages for Debian Bullseye
+  when:
+    - ansible_distribution == "Debian"
+    - ansible_distribution_release == "bullseye"
+  apt:
+    pkg: ['openjdk-11-jre-headless']
+    state: latest
+    update_cache: yes
+  tags:
+    - install
+    - packages
+
+# To avoid issue: https://github.com/georchestra/ansible/issues/55
+# "Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user"
+- name: Install packages for ansible
+  when:
+    - ansible_distribution == "Debian"
+  apt:
+    pkg: ['acl']
+    state: latest
+    update_cache: yes
+  tags:
+    - install
+    - packages
+
 - name: Install packages for Ubuntu
```

Comment:
Ah I see there's an entry for Ubuntu.  The issue is that all our VMs are Debian afaik so the other distros aren't tested and are probably broken.  That's why it's easier to just support Debian and make it a clear requirement.

Reply:
I removed Ubuntu support and show error if it is not Debian